Automattic/jetpack
# AI Assistant: Introduce plugin sidebar

Related with #31946 

## Proposed changes:

* Add new `AiAssistantPluginSidebar` component.
* Add `pluginSettings` and register it under `registerJetpackPlugin`.
* Add `JETPACK_AI_ASSISTANT_PLUGIN_SIDEBAR` flag.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Download `Code Snippets` plugin.
* Introduce a new snippet to activate plugin sidebar: `define('JETPACK_AI_ASSISTANT_PLUGIN_SIDEBAR', true);`
* Go to an post.
* Check if `Jetpack` sidebar has `AI Assistant` panel.
* Click on publish.
* Check if `Pre Publish` sidebar has `AI Assistant` panel.
